src/Bicep.Core.Samples/Files/InvalidResources_CRLF/main.diagnostics.bicep

```diff
@@ -1232,7 +1232,7 @@ resource invalidScope 'My.Rp/mockResource@2020-12-01' = {
 //@[22:53) [BCP081 (Warning)] Resource type "My.Rp/mockResource@2020-12-01" does not have types available. |'My.Rp/mockResource@2020-12-01'|
   name: 'invalidScope'
   scope: notAResource
-//@[9:21) [BCP036 (Error)] The property "scope" expected a value of type "resource" but the provided value is of type "object". |notAResource|
+//@[9:21) [BCP036 (Error)] The property "scope" expected a value of type "resource | tenant" but the provided value is of type "object". |notAResource|

```

Comment:
>tenant [](http://example.com/codeflow?start=85&length=6)

This is expected because we allow all scopes for an unknown resource and the bitmask has changed now.